src/main/java/frc/robot/subsystems/climber/ClimberSubsystem.java

```diff
@@ -30,6 +32,30 @@ public Command manualMoveBackwardCommand() {
         );
     }
 
+    public Command climbSequenceCommand() {
+        return m_deployed ? liftRobotCommand() : deployClimberCommand();
+    }
+
+    public Command deployClimberCommand() {
+        return new FunctionalCommand(
+            () -> m_deployed = true,
+            () -> m_IO.setVoltage(ClimberConstants.AUTO_MOVE_MOTOR_SPEED),
+            (interrupted) -> m_IO.motorOff(),
+            () -> (m_IO.getPosition() > ClimberConstants.DEPLOY_POSITION),
```

Comment:
As is, won't work if you spin the climber around more than once. But I would have the wrapping handled in the IO subsystem.

Also won't work if you spin the climber too far.